src/v/test_utils/http_imposter.h

```diff
@@ -21,8 +21,21 @@ class http_imposter_fixture {
 public:
     static constexpr std::string_view httpd_host_name = "127.0.0.1";
     static constexpr uint httpd_port_number = 4430;
+    static constexpr std::string_view error_payload
+      = R"xml(<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
nit: this error is S3 specific but http_imposter is more general

Reply:
makes sense, I will move it to the service fixture which uses this string.